cephalopod/webinterface/misc.py

```diff
@@ -13,10 +13,19 @@
 from . import bp
 
 
-def get_extra_fields(server_list):
+def get_all_server_fields(server_list):
     extra_fields = set()
     for server in server_list:
-        extra_fields.update(server.crawled_data or {})
+        extra_fields.update(get_server_fields(server) or {})
+    return sorted(extra_fields)
+
+
+def get_server_fields(server):
+    extra_fields = set()
+    fields_settings = current_app.config['CRAWLED_FIELDS_SETTINGS']
+    if server.crawled_data:
+        data = {k: v for k, v in server.crawled_data.iteritems() if fields_settings.get(k, False) is not None}
```

Comment:
I don't follow. The problem is if there would be just `if fields_settings.get(k)` is going to ignore all the fields that are not specified in the config file and this is not the behavior that Cephalopod had previously.

I think there are two approaches we can follow:
1. Taking into account __all__ the crawled fields no matter if they are specified in the config file or not and ignore only the ones that are specified as `None` in the config. In this case the config file would be useful to customize the aggregation functions, chart type, etc. (The approach that I'm following now and quite similar as it was before except the `None` feature)
2. Taking into account __only__ the fields that are specified in the config file and ignore the rest (so the ignore feature wouldn't make sense here)

Not sure if you were referring to this.